cmd/lncli/invoicesrpc_active.go

```diff
@@ -214,9 +218,18 @@ func addHoldInvoice(ctx *cli.Context) error {
 
 	switch {
 	case ctx.IsSet("amt"):
-		amt = ctx.Int64("amt")
+		if ctx.IsSet("amt_msat") {
+			return fmt.Errorf("amt and amt_msat are mutually exclusive")
+		}
+		ms = ctx.Int64("amt") * 1000
+	case ctx.IsSet("amt_msat"):
+		if ctx.IsSet("amt") {
+			return fmt.Errorf("amt and amt_msat are mutually exclusive")
```

Comment:
This is already checked at the RPC level, so we don't need to do so again here.

Reply:
Rewritten following the logic of this comment, so both amt and amt_msat are passed to rpc, and if both are present, the server errors out.